### DISCUSSION TITLE
Intune Management VS Microsoft 365 DSC

### DISCUSSION THREAD

**Discussion by xylphe** [Micke-K/IntuneManagement#283] [General]
Hello !

I'm looking for a solution to backup my Microsoft 365 tenant configuration and I've come across Intune Management and Microsoft 365 DSC.
Are there any noticeable differences in terms of results or are they 2 different ways of obtaining an almost similar result?

Thanks a lot!

**Comment by Micke-K** [comment]
Hello,

I have not used the M365 DSC so I can't comment on full functionality. 

Intune Management is probably an easier way to get started with import and export and it has some additional features like the GUI, Compare, Documentation and a few other tools.
It was created for UI and simple export / import. Automation support has been added along the way, but it is still built on the UI even though running silent. 

DSC is a bit more complex to get started with but a great solution for automation once setup properly. This is probably the best way to automate intent in a single tenant or between tenants. I am saying this without testing it so don't take my word for it. 

What you do should be based on your requirements and what you want to achieve. Using one does not stop you from using the other. I personally use Intune Management for backup and documentation. Sometimes just for quick troubleshooting to see what policies looks like in json format and sometimes to find deployments.
I also use it to automate the documentation and then publish it on an internal web site. So I always have an updated documentation available which makes it very easy to search for a setting and see what policy it is. The documentation documents the policies the exact same way it does in the Intune admin portal. At least for 95+ if all policies. I update the documentation as soon as I get an issue about missing settings or see one myself. The documentation part is the most complex part of the tool since a lot of it cannot be documented via APIs.

Let me know if you have any additional questions.

Cheers!

**Reply by xylphe** [reply]
Hello @Micke-K 

Thank you very much for your very detailed reply!

Have a nice day :)
